kdl_parser/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ target_link_libraries(${PROJECT_NAME} PUBLIC
   orocos-kdl
   urdfdom_headers::urdfdom_headers)
 target_link_libraries(${PROJECT_NAME} PRIVATE
+  rcutils::rcutils
```

Comment:
It's also a good idea to `ament_export_dependencies(rcutils)` in case CMake decides downstream targets need to link against it (which it might do if someone tried to build kdl_parser as a static library).

Reply:
Excellent point.  Done in ed4e01b.